_docs/_latest/configure/scanner/rules.md

```diff
@@ -626,10 +626,155 @@ rules:
+        resource_ids:
+          - org-1
+    applies_to:
+      - 'bucket'
+    locations:
+      - 'us*'
+ - name: All buckets in organization must not be in EU.
+    mode: blacklist
+    resource:
+      - type: organization
+        resource_ids:
+          - org-1
+    applies_to:
+      - 'bucket'
+    locations:
+      - 'eu*'
+```
+
+* `name`
+  * **Description**: The name of the rule.
+  * **Valid values**: String.
+
+* `mode`
+  * **Description**: The mode of the rule.
+  * **Valid values**: One of `blacklist` or `whitelist`.
+
+* `resource`
+  * `type`
+    * **Description**: The type of the resource.
+    * **Valid values**: One of `organization`, `folder` or `project`.
+
+  * `resource_ids`
+    * **Description**: A list of one or more resource ids to match.
+    * **Valid values**: String.
+
+* `applies_to`
+  * **Description**: A list of resource types to apply this rule to.
+  * **Valid values**: Currently only supports `bucket`.
+
+* `locations`:
+  * **Description**: A list of resource locations.
+  * **Value values**: String. Supports wildcards. 
+  * **Note**:
+    * Due to differences in capitalization among resource locations, all resources locations will be lower cased before being matched.
+    * Due to differences in region (europe-west1) vs multi regional (EU) naming, we recommend writing rules that can cover both (e.g. eu* instead of europe*).
+
+## Log Sink rules
+
+### Rule definition
+
+```yaml
+rules:
+  - name: 'Require BigQuery Audit Log sinks in all projects.'
+    mode: required
+    resource:
+      - type: organization
+        applies_to: children
+        resource_ids:
+          - org-1
+    sink:
+      destination: 'bigquery.googleapis.com/*'
+      filter: 'logName:"logs/cloudaudit.googleapis.com"'
+      include_children: '*'
+```
+
+* `name`
+  * **Description**: The name of the rule.
+  * **Valid values**: String.
+
+* `mode`
+  * **Description**: The mode of the rule.
+  * **Valid values**: One of `required`, `blacklist` or `whitelist`.
+
+* `resource`
+  * `type`
+    * **Description**: The type of the resource.
+    * **Valid values**: One of `organization`, `folder` or `project`.
+
+  * `resource_ids`
+    * **Description**: A list of one or more resource ids to match.
+    * **Valid values**: String.
+
+* `applies_to`
+  * **Description**: A list of resource types to apply this rule to.
+  * **Valid values**: One of `self`, `children` or `self_and_children`.
+
+* `sink`
+  * `destination`
+    * **Description**: The destination service. Where the exported log entries will go.
+    * **Valid values**: String.
+
+  * `filter`
+    * **Description**: The logs filter. Determines which logs to export.
+    * **Valid values**: String.
+  
+  * `include_children`
+    * **Description**: Whether to include children. It is only relevant to sinks created for organizations or folders.
+    * **Valid values**: String. One of `true`, `false` or `*`. `*` means use the original sink's config.
```

Comment:
Slight clarification: * means the rule will match sinks with either True or False.

Reply:
Done.